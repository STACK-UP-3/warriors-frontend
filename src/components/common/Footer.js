import React from 'react';

export default () => (
  <footer className='page-footer'>
    <div className='container'>
      <div className='footer'>
        <span>Â© 2020 BareFoot Nomad, All rights reserved.</span>
      </div>
    </div>
  </footer>
);
