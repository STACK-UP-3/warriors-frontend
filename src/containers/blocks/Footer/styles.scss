footer .footer-copyright .row {
	width: 100%;
	margin-bottom: 0;
}

footer .subtitle {
	font-family: 'Archivo', sans-serif;
	font-size: 16px;
	margin-top: 0;
	margin-bottom: 10px;
}

.page-footer .footer-copyright {
	background-color: #083b66;
}

.social-icons a {
	margin-right: 20px;
}
