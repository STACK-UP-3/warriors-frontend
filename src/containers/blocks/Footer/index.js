import React from 'react';
import { Link } from 'react-router-dom';

// Load styles
import './styles.scss';
// Load images
import icoFacebook from '../../../../public/images/icon-facebook.svg';
import icoInstagram from '../../../../public/images/icon-instagram.svg';
import icoSnapchat from '../../../../public/images/icon-snapchat.svg';
import icoGooglePlus from '../../../../public/images/icon-google-plus.svg';

export default class Footer extends React.Component {
  render() {
    return (
      <footer className="page-footer indigo darken-4">
        <div className="footer-copyright">
          <div className="row">
            <div className="col s12 m4 left-align">
              <p className="subtitle">Find Us Online</p>
              <div className="social-icons">
                <a href="#">
                  <img src={icoFacebook} />
                </a>
                <a href="#">
                  <img src={icoInstagram} />
                </a>
                <a href="#">
                  <img src={icoSnapchat} />
                </a>
                <a href="#">
                  <img src={icoGooglePlus} />
                </a>
              </div>
            </div>
            <div className="col s12 m4 valign-wrapper">
              <span id="copyright">
                Copyright Â© 2020 Warriorz Team @ Andela Kigali Stack-Up 3
              </span>
            </div>
            <div className="col s12 m4 right-align">
              <p className="subtitle">Get Customer Support</p>
              <div className="row">
                <div className="col s12 m6"></div>
                <div className="col s12 m6">
                  <Link to="#" className="col s6 m4">
                    <i className="material-icons">call</i>
                  </Link>
                  <Link to="#" className="col s6 m4">
                    <i className="material-icons">sms</i>
                  </Link>
                  <Link to="#" className="col s6 m4">
                    <i className="material-icons">mail</i>
                  </Link>
                </div>
              </div>
            </div>
          </div>
        </div>
      </footer>
    );
  }
}
