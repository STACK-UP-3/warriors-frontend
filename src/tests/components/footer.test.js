import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from '../../components/common/Footer';

describe('Test <Footer/> Component', () => {
  it('Should render Footer with all nodes', () => {
    const footer = shallow(<Footer />);
    expect(toJson(footer)).toMatchSnapshot();
  });
});
