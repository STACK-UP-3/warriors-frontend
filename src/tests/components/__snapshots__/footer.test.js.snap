// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Footer/> Component Should render Footer with all nodes 1`] = `
<footer
  className="page-footer blue darken-3"
>
  <div
    className="container"
  >
    <div
      className="footer"
    >
      <span>
        Â© 2020 BareFoot Nomad, All rights reserved.
      </span>
    </div>
  </div>
</footer>
`;
